src/service/topics.ts

```diff
@@ -1,4 +1,5 @@
 import crashlytics from '@react-native-firebase/crashlytics';
+import moment from 'moment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `moment` library is a good move if you're planning to use it for date/time manipulation. However, ensure that this doesn't introduce unnecessary overhead if the same functionality can be achieved with native JavaScript Date methods.
```diff
+ import moment from 'moment';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->